src/boot/efi/boot.c

```diff
@@ -401,9 +423,10 @@ static VOID print_status(Config *config, CHAR16 *loaded_image_path) {
         console_key_read(&key, 0);
 
         Print(L"timeout:                %u\n", config->timeout_sec);
-        if (config->timeout_sec_efivar >= 0)
+        if (config->timeout_sec_efivar != TIMEOUT_UNSET)
                 Print(L"timeout (EFI var):      %d\n", config->timeout_sec_efivar);
-        Print(L"timeout (config):       %u\n", config->timeout_sec_config);
+        if (config->timeout_sec_config != TIMEOUT_UNSET)
+                Print(L"timeout (config):       %u\n", config->timeout_sec_config);
```

Comment:
these could use some love i guess, i.e. `s` suffix, and handle the two other special cases specially with a pretty string

Reply:
I'd rather keep this here simple and touch it together with print_status() from the other PR...